compiler/x/codegen/OMRAheadOfTimeCompile.hpp

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at http://eclipse.org/legal/epl-2.0
+ * or the Apache License, Version 2.0 which accompanies this distribution
+ * and is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception [1] and GNU General Public
+ * License, version 2 with the OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+#ifndef OMR_X86_AHEADOFTIMECOMPILE_INCL
+#define OMR_X86_AHEADOFTIMECOMPILE_INCL
+
+#ifndef OMR_AHEADOFTIMECOMPILE_CONNECTOR
+#define OMR_AHEADOFTIMECOMPILE_CONNECTOR
+namespace OMR { namespace X86 {  class AheadOfTimeCompile; } }
+namespace OMR { typedef OMR::X86::AheadOfTimeCompile AheadOfTimeCompileConnector; }
+#endif // OMR_AHEADOFTIMECOMPILE_CONNECTOR

```

Comment:
Please add a newline here to add some whitespace so this code isn't so dense.

Reply:
I've introduced some white spaces. Thank you! Will ship that one commit after the next.

Reply:
Resolved now